test/ha/suite.go

```diff
@@ -15,60 +15,21 @@
 package ha
 
 import (
-	"github.com/onosproject/onos-test/pkg/onostest"
-
-	"github.com/onosproject/helmit/pkg/helm"
 	"github.com/onosproject/helmit/pkg/test"
+	"github.com/onosproject/onos-config/test/utils/charts"
 )
 
 type testSuite struct {
 	test.Suite
 }
 
-// TestSuite is the onos-config CLI test suite
+// TestSuite is the onos-config HA test suite
 type TestSuite struct {
 	testSuite
 }
 
-const onosComponentName = "onos-config"
-const testName = "ha-test"
-
-// SetupTestSuite sets up the onos-config CLI test suite
+// SetupTestSuite sets up the onos-config HA test suite
 func (s *TestSuite) SetupTestSuite() error {
-	err := helm.Chart(onostest.ControllerChartName, onostest.AtomixChartRepo).
-		Release(onostest.AtomixName(testName, onosComponentName)).
-		Set("scope", "Namespace").
-		Install(true)
-	if err != nil {
-		return err
-	}
-
-	err = helm.Chart(onostest.RaftStorageControllerChartName, onostest.AtomixChartRepo).
-		Release(onostest.RaftReleaseName(onosComponentName)).
-		Set("scope", "Namespace").
-		Install(true)
-	if err != nil {
-		return err
-	}
-
-	err = helm.Chart("onos-topo", onostest.OnosChartRepo).
-		Release("onos-topo").
-		Set("image.tag", "latest").
-		Set("storage.controller", onostest.AtomixController(testName, onosComponentName)).
-		Install(true)
-	if err != nil {
-		return err
-	}
-
-	err = helm.Chart("onos-config", onostest.OnosChartRepo).
-		Release("onos-config").
-		Set("image.tag", "latest").
-		Set("replicaCount", "2").
-		Set("storage.controller", onostest.AtomixController(testName, onosComponentName)).
-		Install(true)
-	if err != nil {
-		return err
-	}
-
-	return nil
+	umbrella := charts.CreateUmbrellaRelease()
```

Comment:
For HA, we run two instance of onos-config but we are not ready yet to run two instance of onos-config for gnmi tests. Looks like we are using one function for both

Reply:
We should set the values on the returned release then rather than inside `CreateUmbrellaRelease()`.

Reply:
nvm I guess you already did that 🤷

Reply:
nvm I guess we're already doing that, but this should set the replica count to `2`, and the gNMI test should leave it at `1` AIUI